packages/angular-table/src/flexrender.directive.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  Directive,
+  Input,
+  OnInit,
+  TemplateRef,
+  ViewContainerRef,
+} from '@angular/core'
+
+@Directive({
+  selector: '[flexRender]',
+  standalone: true,
+})
+export class FlexRenderDirective implements OnInit {
+  // private vcr = inject(ViewContainerRef)
+  // private templateRef = inject(TemplateRef)
+
+  private _flexRender: any
```

Comment:
We can probably do better than `any`
```suggestion
  private _flexRender: string|() => any|null
```